asyncapi/salesforce/data_types.bal

```diff
@@ -33,8 +37,20 @@ public type ListenerConfig record {|
     string channelName;
     @display{label: "Replay ID", "description": "The replay ID to change the point in time when events are read"}
     int replayFrom = REPLAY_FROM_TIP;
+    @display{label: "Environment", "description": "The type of Salesforce environment"}
+    string environment = PRODUCTION;
```

Comment:
Isn't this a breaking change?

Reply:
This is not breaking change as the existing codes are not affected by the change. As we don't expect PRODUCTION/DEVELOPER users to add the environment they use. This has already being an issue to the SANDBOX users. So can consider as a major change